store/storage/pebbledb/db.go

```diff
@@ -102,6 +136,10 @@ func (db *Database) Has(storeKey string, version uint64, key []byte) (bool, erro
 }
 
 func (db *Database) Get(storeKey string, targetVersion uint64, key []byte) ([]byte, error) {
+	if targetVersion < db.earliestVersion {
+		return nil, nil
```

Comment:
should probably return an error so we can return the error to the user saying the earliest height is x